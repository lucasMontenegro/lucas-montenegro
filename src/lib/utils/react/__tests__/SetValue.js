describe(`lib/utils/react/SetValue`, () => {
  it(`should use the right dependency versions`, () => {
    expect(jestUtils.getDependencies([`react`, `@material-ui/core`])).toMatchSnapshot()
  })
})