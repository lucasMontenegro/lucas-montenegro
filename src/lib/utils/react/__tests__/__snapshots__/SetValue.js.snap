// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lib/utils/react/SetValue should use the right dependency versions 1`] = `
Object {
  "@material-ui/core": "^4.5.0",
  "react": "^16.10.1",
}
`;
